NETWORK/getPhotofromFlicker.playground/Contents.swift

```diff
@@ -0,0 +1,81 @@
+//Get photo(data) from Flicker by location
```

Comment:
Let's try to be more explicit in the file structure and think ahead. 
The plan is to put various snippets from different languages in the repository so thank you very much for your contribution.
In the root direct maybe we should put a more generic folder named something  like  `MOBILE` under this let's put `IOS` or `SWIFT` and finally a specific name for this snippet `GetPhotoFromFlicker`
This will help us to navigate easier. 

So structure will be like
`/MOBILE/SWIFT/GetPhotoFromFlicker/` and your code.

Please remove the NETWORK folder as it doesn't seem to help with navigation.

Reply:
Thanks for the review! I've created a new branch with updates.